clients/go/cmd/zbctl/main_test.go

```diff
@@ -243,6 +243,7 @@ func (s *integrationTestSuite) TestCommonCommands() {
 				goldenOut = fmtGolden
 			}
 
+			fmt.Println(string(cmdOut))
```

Comment:
❓ Is this a development leftover? If so, 🔧  please remove it.

Reply:
Yes, my bad, that's what I get for doing too many things at once